groups/OWNERS

```diff
@@ -10,12 +10,7 @@ options:
 approvers:
 - dprotaso
 - upodroid
-- chizing
+- chizhg
 
 reviewers:
-- MadhavJivrajani
-
-labels:
```

Comment:
Can we use `labels` in any way? If not maybe we could have a comment that this option does exist.

Reply:
need to remove it. we don't use those labels